Trappist/src/Promact.Trappist.Web/wwwroot/rollup-config.js

```diff
@@ -0,0 +1,29 @@
+﻿import nodeResolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import uglify from 'rollup-plugin-uglify';
+
+export default {
+    entry: 'wwwroot/app/main.aot.js',
+    sourceMap: false,
+    format: 'iife',
+    onwarn: function (warning) {
+        // Skip certain warnings
+
+        // should intercept ... but doesn't in some rollup versions
+        if (warning.code === 'THIS_IS_UNDEFINED') { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=javascript%3AOneStatementPerLine)